src/led.c

```diff
@@ -152,8 +153,8 @@ static inline void bl_led__set(
 	 * 2. Making bl_led_init() cache the clear masks so they can be reused
 	 *    here.
 	 */
-	gpio_clear(led_port[port], bl_led__get_pin_mask(port, 0xffff));
-	gpio_set(led_port[port], bl_led__get_pin_mask(port, led_mask));
+	gpio_port_set_masked(led_port[port], bl_led__get_pin_mask(port, 0xffff), '0');
+	gpio_port_set_masked(led_port[port], bl_led__get_pin_mask(port, led_mask), '1');
```

Comment:
You might consider using `gpio_port_set_bits` and `gpio_port_clear_bits` here instead of `gpio_port_set_masked`.
It will do the same thing, just a little clearer.

Reply:
Also, `bl_led__get_pin_mask` returns a `uint16_t`, while the zephyr functions take `gpio_port_pins_t` which is typedef'd to a `uint32_t` so that function would also need changing to return the right type ideally.

Reply:
I beleieve it will be better to keep `gpio_port_set_masked`  because it uses a mask to work with the leds and the function in which this API is called has a mask as an input argument. Therefore, not using a mask would mean  more work refactoring the wrapping function.

Reply:
About using `0` instead of `0x00`, I did that because I noticed that in some tests where doing it to: 
https://github.com/zephyrproject-rtos/zephyr/blob/45418dedc0bde40d69b61dbfa6c0449f1098aba8/tests/drivers/gpio/gpio_basic_api/src/test_gpio_port.c#L569 

However I sure did the mistake in the `1` section.

Reply:
I just double checked and it seems like `1` and `0` should be fine.
https://docs.zephyrproject.org/latest/reference/peripherals/gpio.html#c.gpio_port_set_masked